frontend/src/features/mapSideBar/layout/MapSideBarLayout.tsx

```diff
@@ -1,115 +1,108 @@
 }) => {
   return (
-    <StyledMapSideBarLayout
-      className={classNames('map-side-drawer', {
-        show: show,
-      })}
-    >
+    <StyledMapSideBarLayout show={showSideBar}>
       <VisibilitySensor partialVisibility={true}>
         {({ isVisible }: any) => (
           <>
             <TitleBar>
-              <Underline>
-                <LotIcon className="mr-1" />
-                <Styled.H1 className="mr-auto">{title}</Styled.H1>
+              <Row>
+                <Col>
+                  <Styled.H1 className="mr-auto">
+                    {icon}
+                    {title}
+                  </Styled.H1>
+                </Col>
 
-                <TooltipWrapper toolTipId="close-sidebar-tooltip" toolTip="Close Form">
-                  <CloseIcon title="close" onClick={() => setShowSideBar(false)} />
-                </TooltipWrapper>
-              </Underline>
+                {showCloseButton && (
+                  <Col xs="auto">
+                    <TooltipWrapper toolTipId="close-sidebar-tooltip" toolTip="Close Form">
+                      <CloseIcon title="close" onClick={() => setShowSideBar(false)} />
+                    </TooltipWrapper>
+                  </Col>
+                )}
+              </Row>
+              <Underline />
             </TitleBar>
-            <Header>{props.header}</Header>
-            <Content>{isVisible ? props.children : null}</Content>
+
+            <StyledBody>
+              {header && isVisible && <Header>{header}</Header>}
+              <Content>{isVisible ? props.children : null}</Content>
+            </StyledBody>
           </>
         )}
       </VisibilitySensor>
     </StyledMapSideBarLayout>
   );
 };
 
-const Content = styled.div`
-  grid-area: content;
+const StyledBody = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
   width: 100%;
   height: 100%;
-  top: 2rem;
-  position: absolute;
-  margin-top: 1rem;
+  position: relative;
 `;
 
-const Header = styled.div`
-  grid-area: header;
+const Content = styled.div`
+  height: 100%;
+  width: 100%;
 `;
 
-const LotIcon = styled(LotSvg)`
-  width: 3rem;
-  height: 3rem;
-  align-self: flex-end;
-`;
+const Header = styled.div``;
 
-const TitleBar = styled.div`
-  grid-area: title;
-  display: flex;
-`;
+const TitleBar = styled.div``;
 
 const Underline = styled.div`
   width: 100%;
-  display: flex;
   border-bottom: solid 0.5rem ${props => props.theme.css.primaryLightColor};
 `;
 
-const StyledMapSideBarLayout = styled.div`
+const StyledMapSideBarLayout = styled.div<{ show: boolean }>`
   h1 {
     border-bottom: none;
+    margin-bottom: 0.2rem;
   }
-  height: calc(
-    100vh - ${props => props.theme.css.headerHeight} - ${props => props.theme.css.footerHeight}
-  );
-  min-width: 93rem;
-  margin-left: -93rem;
-  max-width: 93rem;
-  padding: 2.4rem 3.6rem;
-  display: grid;
-  grid: 4.2rem 5.8rem 1fr / 1fr;
-  grid-template-areas:
-    'title'
-    'header'
-    'content';
```

Comment:
This was necessary since the child layout was being pushed out of view if there was anything on the 'header'. Without it, the extra calculations are not necessary since the view is not absolute anymore.

Reply:
This has led to a regression in how the sidebar is animated, the forms now squish/unsquish as the sidebar is closed.

Reply:
Also the screen is cut off:

![image](https://user-images.githubusercontent.com/41091511/162852629-9766d826-3004-449a-8b10-0725c215084e.png)